packages/core/src/lib/util.ts

```diff
@@ -224,3 +225,23 @@ export function validatePackageId(input: string): string | null{
   }
   return null;
 }
+
+/**
+ * Removes a file or directory. If the path is a directory, recursively deletes files and
+ * directories inside it.
+ */
+export async function rmdirs(path: string): Promise<void> {
```

Comment:
Shouldn't this be called `rmdir` instead of `rmdirs`? I would assume `rmdirs` would take a list of paths.

Reply:
Renamed.